src/common/utils.js

```diff
@@ -72,7 +72,7 @@ module.exports = {
     return 21000 +
       hubOverhead +
       calldataSize * gtxdatanonzero +
-      relayCallGasLimit +
+      parseInt(relayCallGasLimit) +
```

Comment:
I thnk the field members can be defined as ints. no reason to parseInt them.

Reply:
All of these will be fixed once utils are translated to TypeScript. This will require server to be compiled as well. Now I am afraid to change types of fields because instead of compilation errors it may cause hard-to-find bugs in server where, in JS logic, `'1' + 2 = '12'`